src/grz_cli/upload.py

```diff
@@ -23,8 +23,7 @@
 from .progress_logging import FileProgressLogger
 from .states import UploadState
 
-MULTIPART_THRESHOLD = 8 * 1024 * 1024  # 8MiB, boto3 default
-MULTIPART_CHUNKSIZE = 8 * 1024 * 1024  # 8MiB, boto3 default
+MULTIPART_THRESHOLD = 8 * 1024**2  # 8MiB, boto3 default, largely irrelevant
```

Comment:
Why? :slightly_smiling_face:

Reply:
Whoops, did not intend to keep that. But this is the threshold at which boto3 will even consider doing multipart uploads. As long as this is not in the order of bytes, this will not have much of an impact.